free-courses-en.md

```diff
@@ -114,6 +115,11 @@
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
 
+### Bootstrap
+
+* [Rapid website design with Bootstrap](https://www.udemy.com/course/responsive-website-design) - Laurence Svekis (Udemy)
```

Comment:
Could you please write the title of the resource in title case, like in the actual webpage?

Reply:
It's common for library catalogs to capitalize differently. Proper nouns, though, like JavaScript should follow the resource - I know, it's obscure, but we need to pick our battles.